subworkflows/local/utils_nfcore_mcmicro_pipeline/main.nf

```diff
@@ -221,6 +221,29 @@ def validateInputMarkersheet( markersheet_data ) {
         error("Duplicate [channel, cycle] pairs: ${dups}")
     }
 
+    // validate backsub columns if present
+    def exposure_list = markersheet_data.findResults{ _1, _2, _3, _4, _5, _6, exposure, _8, _9 -> exposure != [] ? exposure : null }
```

Comment:
Yes, that's much cleaner.  Thank you Jeremy!
I kept `null` instead of `false` because that allows the lists to shrink and I'm checking the cases below when they're empty.
I tried using `false`, but it doesn't shrink the list, so I think it would require more refactoring.
Please reopen if I'm not understanding this. Thanks!